PowerShell.Common.props

```diff
@@ -94,7 +94,7 @@
     <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
 
     <TargetFramework>netcoreapp2.0</TargetFramework>
-    <RuntimeFrameworkVersion>2.0.4-servicing-25831-01</RuntimeFrameworkVersion>
```

Comment:
should we update the appveyor yml to flush the cache?

Reply:
Good point! Fixed.